catchup/catchpointService.go

```diff
@@ -453,7 +453,7 @@ func (cs *CatchpointCatchupService) processStageBlocksDownload() (err error) {
 	blocksFetched := uint64(1) // we already got the first block in the previous step.
 	var blk *bookkeeping.Block
 	var client FetcherClient
-	for attemptsCount := uint64(1); blocksFetched <= lookback; attemptsCount++ {
+	for attemptsCount := uint64(1); blocksFetched <= lookback; {
```

Comment:
Since this is only counting the failed attempts, can you call it failedAttemptsCount or retryCount instead of attemptsCount?

Reply:
Calling it "retryCount" makes sense.